ncbi/src/main/java/org/reactome/release/EuropePMC.java

```diff
@@ -0,0 +1,375 @@
+package org.reactome.release;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.neo4j.driver.v1.Session;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.LinkedHashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.reactome.release.Utilities.appendWithNewLine;
+
+/**
+ * File generator for Europe PMC.  This class will produce files for
+ * Europe PMC profile XML describing Reactome as a data provider and
+ * Europe PMC link XML describing Reactome pathways annotated directly
+ * (i.e. not on parent or child pathways) with PubMed literature references.
+ * @author jweiser
+ */
+public class EuropePMC {
+	private static final Logger logger = LogManager.getLogger();
+	private static final String rootTag = "links";
+	private static final int reactomeProviderID = 1903;
```

Comment:
What is this?

Reply:
This is the identifier Europe PMC has given to Reactome so they have an id to associate with our submissions.